development/sass/top-nav.scss

```diff
@@ -0,0 +1,60 @@
+$nav-fonts: "Helvetica Neue",Helvetica,Arial,sans-serif;
```

Comment:
move this file into a folder call "modules" so it will be located in 
`development/sass/module/_top-nav.scss`